app/assets/javascripts/utils/language_names.js

```diff
@@ -729,14 +893,35 @@ const LanguageNames = {
       name: 'Zhuang',
       nativeName: 'Saɯ cueŋƅ',
     },
+    zgh: {
+      name: 'Standard Moroccan Tamazight',
+      nativeName: 'ⵜⴰⵎⴰⵣⵉⵖⵜ',
+    },
     zh: {
       name: 'Chinese',
       nativeName: '中文',
     },
+    'zh-CN': {
+      name: 'Simplified Chinese',
+      nativeName: '中文（简体）',
+    },
+    'zh-hans': {
+      name: 'Simplified Chinese',
+      nativeName: '中文（简体）‎',
+    },
+    'zh-hant': {
+      name: 'Traditional Chinese',
+      nativeName: '中文（繁體）‎',
+    },
+    'zh-TW': {
+      name: 'Traditional Chinese',
+      nativeName: '中文（繁體）‎',
```

Comment:
It looks like zh-TW and zh-hant use the same name and native name, and the same is true for zh-CN and zh-hans. However, they have separate translation files (and separate sets of translators).

But even in the comments from translatewiki.net, the language code is the only thing that shows they are different. Both zh-hant and zh-TW say "Messages for Traditional Chinese (中文（繁體）‎)", for example.

After this change, the language picker component will show two entries in a row for `中文（繁體）` and for `中文（简体）`.  I would love some advice from @amire80 or @Nikerabbit or another i18n expert on this. I guess for most people it won't be a problem, because the language code gets selected via browser preferences, so users from Taiwan should get zh-TW by default.

There are a few other cases with the same native name, such "ug-arab" vs "ug", but in those cases, we only actually have one of the versions available (only ug-arab, for example), so it won't cause any problems.

Reply:
I used the names from the comments you mentioned but I crosschecked them with data on the Wikipedia. Regarding the Chinese names, I have found right now this page https://en.wikipedia.org/wiki/Chinese_Wikipedia where the names are described better, I hope. Shall I replace them?

Reply:
@jerry-74 I guess you mean the names from the table in the Solutions section? That seems like a good solution to me. I'm sure someone will report it if it doesn't make sense. Please do replace those, and then I'll merge.

Reply:
I guess those names from the table correspond to the part in parentheses? So cn-TW would be `中文（台灣正體）`, and so on, I think. That seems consistent with how Google Translate interprets it: Chinese (Taiwanese).

Reply:
Done. See my last commit. Hoping it will be ok.